src/status_im2/contexts/communities/home/view.cljs

```diff
@@ -11,10 +11,7 @@
   [id]
   (let [community-item (rf/sub [:communities/home-item id])]
     [quo/communities-membership-list-item
-     {:on-press      (fn []
-                       (rf/dispatch [:communities/load-category-states id])
-                       (rf/dispatch [:dismiss-keyboard])
-                       (rf/dispatch [:navigate-to-nav2 :community {:community-id id}]))
+     {:on-press      #(rf/dispatch [:navigate-to :community-overview id])
```

Comment:
dismiss-keyboard removed intentionally?

Reply:
yes, I believe it is a leftover from old pages where the "find" section was present and the user could have a keyboard opened when opening community. Not really needed here